recipes/mpdecimal/2.5.x/patches/4.0.0-0001-msvc-fixes.patch

```diff
@@ -0,0 +1,97 @@
+diff --git a/libmpdec++/Makefile.vc b/libmpdec++/Makefile.vc
+index 5985c33..bc2bdac 100644
+--- a/libmpdec++/Makefile.vc
++++ b/libmpdec++/Makefile.vc
+@@ -14,11 +14,11 @@ LIBIMPORT_CXX = libmpdec++-4.0.0.dll.lib
+ LIBSHARED_CXX = libmpdec++-4.0.0.dll
+ 
+ !if "$(DEBUG)" == "1"
+-OPT = /MTd /Od /Zi /EHsc
+-OPT_SHARED = /MDd /Od /Zi /EHsc
++OPT = /Od /Zi /EHsc
++OPT_SHARED = /Od /Zi /EHsc
+ !else
+-OPT = /MT /O2 /GS /EHsc /DNDEBUG
+-OPT_SHARED = /MD /O2 /GS /EHsc /DNDEBUG
++OPT = /O2 /GS /EHsc /DNDEBUG
++OPT_SHARED = /O2 /GS /EHsc /DNDEBUG
+ !endif
+ 
+ !if "$(CC)" == "clang-cl"
+@@ -27,9 +27,9 @@ WARN = /W4 -Wno-undefined-inline
+ WARN = /W4
+ !endif
+ 
+-MPD_CXXFLAGS = $(WARN) /nologo $(OPT)
+-MPD_CXXFLAGS_SHARED = /DBUILD_LIBMPDECXX $(WARN) /nologo $(OPT_SHARED) $(PGOFLAGS)
+-MPD_BIN_CXXFLAGS_SHARED = $(WARN) /nologo $(OPT_SHARED) $(PGOFLAGS)
++MPD_CXXFLAGS = $(WARN) /nologo $(OPT) $(CONAN_CXXFLAGS) $(CONAN_CFLAGS)
++MPD_CXXFLAGS_SHARED = /DBUILD_LIBMPDECXX $(WARN) /nologo $(OPT_SHARED) $(PGOFLAGS) $(CONAN_LDFLAGS)
++MPD_BIN_CXXFLAGS_SHARED = $(WARN) /nologo $(OPT_SHARED) $(PGOFLAGS) $(CONAN_LDFLAGS)
```

Comment:
Not sure if this section is still necessary, is this 1.x legacy? If so, I can remove from previous versions too.

Reply:
I'm removing these sections in my followup PR